src/index.ts

```diff
@@ -1,13 +1,21 @@
 import { ApolloServer } from "apollo-server-express";
+import bodyParser from "body-parser";
 import express from "express";
 import mongoose from "mongoose";
 
 import { MONGO_URL } from "./constants";
 import resolvers from "./resolvers";
 import typeDefs from "./schema";
+import { slackQuery } from "./utils";
 
 const startServer = async () => {
   const app = express();
+  app.use(bodyParser.urlencoded({ extended: true }));
+
+  // may need to change to GraphQL query
```

Comment:
Remove.

Reply:
I think we need line 13 in order to parse Slack's request, but we've replaced it with the updated version:
app.use(express.urlencoded({ extended: true }))